src/citrine/informatics/executions/generative_design_execution.py

```diff
@@ -0,0 +1,114 @@
+# from deprecation import deprecated
+from functools import partial
+from typing import Optional, Iterable
+from uuid import UUID
+
+from citrine._rest.asynchronous_object import AsynchronousObject
+from citrine._rest.pageable import Pageable
+from citrine._rest.paginator import Paginator
+from citrine._rest.resource import Resource
+from citrine._serialization import properties
+from citrine._session import Session
+from citrine._utils.functions import format_escaped_url
+from citrine.informatics.generative_design import GenerationExecutionResult
+
+
+# TODO: This is not pretty, but I needed something to hold the status
+# When we build the GenerativeDesignExecution.
+# This and GenerationResult at `generative_design.py` are the same situation.
+# Is there a way to avoid this?
+class GenerativeExecutionStatus:
```

Comment:
@anoto-moniz This part of the code is iffy. The status I get from the api is a dict {"minor": str, "info": list(str)}. So Without this "status object" I wasn't sure how to correctly build the GenerativeDesignExecution. Happy to figure out what the solution is here.

Reply:
With Sean's status change yesterday, you'll want to update this to match the other executions.

Reply:
And once you do, it should fix the broken tests.

Reply:
Excelling This and a couple other small things on the way!